roles/pre-ansimail/files/ansimail_scripts/helpers/check-user

```diff
@@ -6,16 +6,16 @@
 
 . /usr/local/lib/ansimail/helpers/variables
 
-user=$1
+user="$1"
```

Comment:
I just noticed that there is a weird bug in this script, that is fortunately mitigated due to local scoping and overloading of variables.  
Bug: `user` is defined as a global variable and is also the local iterator variable of the for loop.
Fix: change `user` --> `username` in line 9 and change `$1` to `$username` in line 18